doc/themes/scikit-learn-modern/static/css/theme.css

```diff
@@ -998,6 +998,20 @@ div.sphx-glr-thumbcontainer {
   word-wrap: break-word;
 }
 
+/* authors */
+.authors-container {
```

Comment:
I have been using a `sk-*` convention to make it easier to know that classes we add:

```suggestion
.sk-authors-container {
```

Reply:
Fixed in 7a02b92b2128352fbbb85eb5e14488c9f15b6531 (together with all similar issues below).